interfaces/seccomp/template.go

```diff
@@ -480,6 +480,7 @@ socket AF_NFC
 socket AF_VSOCK
 socket AF_MPLS
 socket AF_IB
+socket AF_QIPCRTR
```

Comment:
we add support for both AF_QIPCRTR and PF_QIPCRTR to the resolver, can we also add PF_QIPCRTR here too?

Reply:
`seccomp/template.go` doesn't declare any PF_*, and AF_X is generally the same as PF_X.
So I think it's not necessary to declare PF_*?

Reply:
sure I guess I didn't notice we only use the `AF_` here and not the `PF_` variants, this is fine then